packages/eslint/package.json

```diff
@@ -27,16 +27,37 @@
     "requirements": {},
     "migrations": "./migrations.json"
   },
+  "exports": {
```

Comment:
How come this was needed?

Reply:
I based it off of @AgentEnder's PR assuming that is intended direction.

I will revert this block entirely